test/mocha/content/content.test.js

```diff
@@ -1,51 +1,56 @@
-'use strict';
 
 const assert = require('assert');
 const app = require('../../../app');
 const chai = require('chai');
-const expect = chai.expect;
+
+const { expect } = chai;
 const chaiHttp = require('chai-http');
+const { logger } = require('handlebars');
+const { logError } = require('gulp-sass');
```

Comment:
Codacy found an issue: ['logError' is assigned a value but never used.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-client/pullRequest?prid=6045527)